newuniquetechstack.md

```diff
@@ -0,0 +1,19 @@
+# Technologie
+Om invulling te geven aan de uitgangspunten in het [manifest] zal er technologie beschikbaar moeten komen. Deze hoeft niet perse vanaf nul opgebouwd te worden. Als er gebruik kan worden gemaakt van reeds bestaande oplossingen, dan is dat mooi meegenomen. Uiteraard zal zowel nieuwe als bestaande technologie moeten voldoen aan de gestelde uitgangspunten. 
+Maar over welke technologie hebben we het hier eigenlijk? En wat is er minimaal nodig om een decentraal communicatienetwerk op te zetten?
+
+## Less is more
+Op dit moment is onze opvatting dat de gedeelde infrastructuur qua componenten zo klein mogelijk moet zijn. Het uitgangspunt hier is dus niet volledigheid, maar wat is minimaal noodzakelijk. Alles wat niet voor iedere gebruikers use-case relevant is kan wegblijven. Wat dan overblijft is de kern. En die kern zal bestaan uit componenten die altijd nodig zijn. Ongeacht welk type oplossing je als softwareleverancier voor ogen hebt om te gaan maken.
+
+## The big 4
+Er zijn 4 kerncomponenten die samen het Nuts netwerk vormen, namelijk **identificatie**, **toestemming**, **adressering** en **logging**.
+
+### Identificatie
+Als je toegang wilt tot gegevens van jezelf of anderen, dan zal je je als eerste moeten identificeren. In de fysieke wereld is het gebruikelijk om je paspoort of rijbewijs te laten zien. In de digitale wereld gaat dat net even anders. Het is gebruikelijk om in te loggen met gebruikersnaam en wachtwoord of middels een identiteitsleverancier?? (identityprovider). Er is echter meteen een duidelijk verschil te zien. Je paspoort is namelijk aan jou afgegeven door een instantie die we met z'n allen vertrouwen, namelijk de gemeente. En degene die de identiteit beheert ben jij zelf. 
+In de digitale wereld is jouw gebruikersnaam afgegeven door (de applicatiebeheerder van) de applicatie waar je gebruik van wilt maken. Vervolgens ligt het beheer van jouw identiteit ook daar en zul je deze partij of applicatiebeheerder moeten vertrouwen. En daar zit meteen het probleem, want deze persoon ken je helemaal niet. En kan je die partij die jouw identiteit beheert eigenlijk wel vertrouwen?  
```

Comment:
De huidige digitale wereld? Of traditionele software applicaties? Misschien wel mooier om te beginnen met de echte wereld, en dan laten zien dat dit anders en onlogisch is geregeld in software.